pkg/storaged/pages.jsx

```diff
@@ -814,16 +814,16 @@
     }
 
     return (
-        <Page id="storage">
+        <Page id="storage" className={client.in_anaconda_mode() ? "anaconda-mode" : ""}>
             { parent_crumbs.length > 0 &&
-            <PageBreadcrumb stickyOnBreakpoint={{ default: "top" }}>
+            <PageBreadcrumb stickyOnBreakpoint={client.in_anaconda_mode() ? {} : { default: "top" }}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19876-20240125-115723-d7fb71dd-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)